File: src/main/java/io/sch/historyscan/domain/contexts/analysis/common/CodeBaseCommit.java
package io.sch.historyscan.domain.contexts.analysis.common;

import io.sch.historyscan.domain.contexts.analysis.history.CodeBaseHistoryCommitFile;
import io.sch.historyscan.domain.contexts.analysis.history.CodeBaseHistoryCommitInfo;

import java.util.List;

public record CodeBaseCommit(CodeBaseHistoryCommitInfo info, List<CodeBaseHistoryCommitFile> files) {
}

File: historyscan-back/src/main/kotlin/io/sch/historyscan/domain/contexts/analysis/common/CodeBaseCommit.kt
package io.sch.historyscan.domain.contexts.analysis.common

import io.sch.historyscan.domain.contexts.analysis.history.CodeBaseHistoryCommitFile
import io.sch.historyscan.domain.contexts.analysis.history.CodeBaseHistoryCommitInfo

class CodeBaseCommit(val info: CodeBaseHistoryCommitInfo, val files: List<CodeBaseHistoryCommitFile>) {
}
